recipes/pretextsnapshot/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "PretextSnapshot" %}
+{% set version = "0.0.2" %}
+{% set checksum = "bbb4ce292c13263616b24233f49859cb9d139cd84d56729ec5dd495782d87536" %}
+{% set about = "Commandline image generator for Pretext contact maps." %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/wtsi-hpag/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} # [linux64]
+    - make
+    - clang>=9.0.0
+    - python
```

Comment:
Can you add libdeflate in as a dependency? There's no need to build it again.

Reply:
I build a series of sub-executables with different hardware support (AVX2, SSE4.2 etc) and I would prefer to build the libraries I use accordingly.